sdcm/tester.py

```diff
@@ -669,6 +669,7 @@ def set_system_auth_rf(self):
             self.log.info('repair system_auth keyspace ...')
             node.run_nodetool(sub_cmd="repair", args="-- system_auth")
 
+    @lru_cache(maxsize=None)
```

Comment:
I don't understand how it solves the original issue.
Please explain it in the commit message.

Reply:
This function creates the tables needed to run alternator tests. The problem is that the method is called several times during the rolling upgrade test. 
To avoid this problem does not happen again, I change the method to "call" one time in the session.

Reply:
Simply looking at it, i think it is not so readable and understood if there are multiple calls to a "create-table" method and the reader has to realize it is cached. so the optimal would be to make sure there is only one call. The caching may be a nice idea in order to still always be on the safe side. BTW - why using @lru_cache and not just @cache ?

Reply:
This problem has occurred in the past.
I prefer to be on the "safe side" and limit the function's call in the run.
I do not know `@cache` in Python, maybe you meant `@lru_cache`?

I can use `cached_property` and convert the method to property.

Reply:
Thanks, it is new decorator in Python